bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -696,26 +696,73 @@ sitemap netatmo label="Netatmo" {
 }
 ```
 
+## Rule Actions
 
-# Sample data
+Multiple actions are supported by this binding. In classic rules these are accessible as shown in this example (adjust getActions with your ThingId):
+
+Example
+
+```
+ val actions = getActions("netatmo","netatmo:room:home:livingroom")
+ if(null === actions) {
+        logInfo("actions", "Actions not found, check thing ID")
+        return
+ }
+```
+
+### setThermRoomTempSetpoint(temp,endtime)
+
+Sends a temperature setpoint (and switch to manual mode) to the thermostat for a room with an end time.
+
+Parameters:
+
+| Name    | Description                                                          |
+|---------|----------------------------------------------------------------------|
+| temp    | The temperature setpoint                                             |
+| endtime | Time the setpoint should end                                         |
+
+Example:
+
+```
+actions.setThermRoomTempSetpoint(19.0,240)
+```
+
+### setThermRoomModeSetpoint(mode,endtime)
+
+Sends a mode to the thermostat for a room with an optional end time.
+
+Parameters:
+
+| Name    | Description                                                          |
+|---------|----------------------------------------------------------------------|
+| mode    | The mode to set: MANUAL, SCHEDULE or FG (Frost-Guard)                |
+| endtime | Time the setpoint should end                                         |
+
+Example:
+
+```
+actions.setThermRoomModeSetpoint("HOME",null)
```

Comment:
Additional example to be provided when the unit of the endtime parameter is clarified.